### DISCUSSION TITLE
Shogun

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#73] [Card Balance]
Shogun is moderate warrior, but his skill is redudant.

Shogun (MISHIMA)
SAMURAI 
F: 5 S: 4 A: 6 V: 6
❌ With Scalper Equipped: [F] +2, opponent's [F] -2. 
➡️ DUALWIELD BLADES

**Comment by 21-ric** [comment]
Awesome idea!

**Comment by david-roxygen** [comment]
The most important thing in my opinion is to remove the ability based on specific card (i.e. with a scalper).
Maybe we could change ' scalper' with a more generic 'blade'.

Shogun (MISHIMA)
SAMURAI
F: 5 S: 4 A: 6 V: 6
With BLADE Equipped: [F] +2, opponent's [F] -2.
DUALWIELD BLADES

the suggested version looks very nice.

Shogun (MISHIMA)
SAMURAI
F: 5 S: 4 A: 6 V: 6
DUALWIELD BLADES

could be equipped with katana e dai-sho or katana and scalper
